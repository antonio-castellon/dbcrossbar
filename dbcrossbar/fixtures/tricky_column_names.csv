"person__Delivery Zone 4.14","person__$Region - 8/1","person__Do you have any allergies?"
a,b,c
