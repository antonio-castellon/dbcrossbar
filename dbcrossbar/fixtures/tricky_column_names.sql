CREATE TABLE tricky_column_names (
  "person__Delivery Zone 4.14" text,
  "person__$Region - 8/1" text,
  "person__Do you have any allergies?" text,
  "name" text,
  "Name" text
)
