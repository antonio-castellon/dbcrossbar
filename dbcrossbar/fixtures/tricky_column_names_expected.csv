person__delivery_zone_4_14,person___region___8_1,person__do_you_have_any_allergies_,name,name_2
a,b,c,d,e
